### DISCUSSION TITLE
How may I contribute to a new language locale?

### DISCUSSION THREAD

**Discussion by purybr365** [Web3Auth/Web3Auth#903] [Feature Requests]
Hi all!

We're building a new product at https://usepicnic.com and using web3auth as a wallet provider. ;)

We would love to contribute to a pt-BR locale. How may we help in that?

**Comment by shahbaz17** [comment]
Hey @purybr365 

We have an open source project https://github.com/web3auth/web3auth-locales where you can add your localized/translated strings.

**Comment by purybr365** [comment]
I just sent a PR to include a Portuguese translation to Web3Auth and Openlogin. Hope you'll approve it.

;)

**Reply by purybr365** [reply]
Perhaps I should mention the one committing the most here: @chaitanyapotti
